.build_web.py

```diff
@@ -20,14 +20,25 @@ def build_web():
         os.chdir("web")
         print("Attempting to build webpage...")
         try:
-            print check_output(["npm", "install"])
-            print check_output(["node_modules/.bin/gulp"])
+            if platform.system() == "Windows":
+                print check_output(["npm.cmd", "install", "--only=dev"])
+                print check_output(["node_modules\\.bin\\gulp.cmd"])
+            else:
+                print check_output(["npm.cmd", "install"])
+                print check_output(["node_modules/.bin/gulp"])
             copyfile("build/index.html.gz.h", "../dist/index.html.gz.h")
-
+        except OSError as e:
+            print "Encountered error OSError building webpage: ", e
+            if e.filename:
+                print "Filename is", e.filename
```

Comment:
not that it matters much, but should there be a space at the end of the string here?